### ISSUE TITLE
update documentation of deployment for ghpages

### ISSUE THREAD

**Issue by AmitPress** [lektor/lektor#1011]
I am struggling to deploy the app in ghpages. As travis ci is now paid, the video the author made is now obsolete. It will be very nice if there is any updated documentation for deployment.

**Comment by dairiki** [comment]
I'm guessing this is the page your referring to: [Travis CI](https://www.getlektor.com/docs/deployment/travisci/)?

And, you're right, it should be updated to describe how to do the same thing using a [GitHub workflow](https://docs.github.com/en/actions/using-workflows).  (If you'd like to help, pull-requests at [lektor-website](https://github.com/lektor/lektor-website/) are welcome.) 

We're currently reworking the deployment process for the Lektor website.  It might help answer some questions to look over the workflow that deploys the Lektor website to GitHub pages: [lektor-website/.github/workflows/deploy.yml](https://github.com/lektor/lektor-website/blob/master/.github/workflows/deploy.yml).

(Currently, this is deploying to https://lektor.github.io/lektor-website/ — we're still working on getting DNS updated to make that available at getlektor.com.)

**Comment by AmitPress** [comment]
Yeah thanks for your input it helped me a lot. I successfully published my blog using github actions and made a PR explaining how I did that.  I will close this issue within 6 hours as for me it is kind of solved.
